src/guides/v2.4/config-guide/cli/config-cli-subcommands-catalog-sync.md

```diff
@@ -0,0 +1,26 @@
+---
+group: configuration-guide
+title: Catalog Sync
+---
+
+Magento compiles catalog data into tables using indexers. This process is automatically [triggered by several events](https://docs.magento.com/user-guide/system/index-management-events.html), such as changes to a product's price or inventory levels. To allow some Magento services to use that catalog data, a catalog sync process runs hourly. The catalog sync process exports product data from the Magento server to SaaS services on an ongoing basis. For example, the [Product Recommendations](https://docs.magento.com/user-guide/marketing/product-recommendations.html) feature needs up-to-date catalog information so that it can accurately return recommendations with correct names, pricing, and availability. In addition to providing a [catalog sync dashboard](https://docs.magento.com/user-guide/system/catalog-sync.html) in the Admin UI to observe and manage the Magento to SaaS syncing process, you also can use the command-line interface to trigger the catalog sync and reindex the products in the SaaS service. This is especially useful if you notice your catalog data is not syncing correctly using the Catalog Sync dashboard.
```

Comment:
```suggestion
Magento compiles catalog data into tables using indexers. This process is automatically [triggered by several events](https://docs.magento.com/user-guide/system/index-management-events.html), such as changes to a product's price or inventory levels. To allow some Magento services to use that catalog data, a catalog sync process runs hourly. The catalog sync process exports product data from the Magento server to SaaS services on an ongoing basis. For example, the [Product Recommendations](https://docs.magento.com/user-guide/marketing/product-recommendations.html) feature needs up-to-date catalog information so that it can accurately return recommendations with correct names, pricing, and availability. In addition to providing a [catalog sync dashboard](https://docs.magento.com/user-guide/system/catalog-sync.html) in the Admin UI to observe and manage the Magento to SaaS syncing process, you also can use the command-line interface to trigger the catalog sync and reindex product data for consumption by SaaS services. This is especially useful if you notice your catalog data is not syncing correctly using the Catalog Sync dashboard.
```